db/migrate/20161129222053_polymorphize_quotes.rb

```diff
@@ -0,0 +1,8 @@
+class PolymorphizeQuotes < ActiveRecord::Migration
+  def change
+    rename_column :quotes, :anime_id, :media_id
+    add_column :quotes, :media_type, :string, null: false
```

Comment:
So I can't do it with something like this?
`add_column :quotes, :media_type, :string, default: 'Anime', null: false`